xml/operating_system_darwin.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Patterns for common names of Mac OS X operating systems.
+-->
+<fingerprints matches="operating_system.name">
+  <!-- Match Mac OS Classic first due to weak matching on Mac OS X -->
+  <fingerprint pattern="^(?:Apple )?Mac OS ([7-9](?:\.\d+?)*?)$" flags="REG_ICASE">
```

Comment:
Anything that uses Recog's interfaces (metasploit), yes, that will work.  For projects that utilize Recog's data (nexpose),  I'm not sure at this point.  Probably best to run this by @pmiseiko-r7 internally.